node.go

```diff
@@ -80,6 +80,33 @@ func (node *Node) run(db *DB) {
 	}
 }
 
+func dataMapToDataArray(m map[int]Data) []Data {
+	i := 0
+	a := make([]Data,len(m))
+	for _,val := range m {
+		a[i] = val
+		i += 1
+	}
+	return a
+}
+
+func consolidateDataVals(vals []Data) []Data {
```

Comment:
Returns an array of Data vals with the highest clock values associated to their respective nodes